content/en/actions/datastore/create.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Create and Manage Datastores
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+- link: "https://www.datadoghq.com/blog/datadog-datastore/"
+  tag: "Blog"
+  text: "Enhance your automated workflows and apps with Datastore"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">App Builder is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+You can create and manage datastores from the [Datastore page][1].
+
+## Create a datastore
+
+To create a datastore:
+
+1. Navigate to the [Datastore page][1].
+1. Click **+ New Datastore**.
+1. Enter a **Name** for your datastore.
+1. Enter a **Primary Key** or toggle the option to **Autogenerate a Primary Key** if a primary key is not essential to your use case.
+   - If you choose to enter a primary key, the key must be a column name in your data where each key has a unique value.
+   - Choosing to Autogenerate a key removes your ability to provide your own keys for new items in the datastore, but you can still update specific keys on existing items.
```

Comment:
@wfa207 Can you check the wording here to make sure it's correct?

Reply:
Can we change to:

```suggestion
   - Choosing to Autogenerate a key removes your ability to provide your own keys for new items in the datastore, but you can still update existing items by specifying their specific keys.
```